### PR TITLE
added gapii target for arm linux (#303)

### PR DESCRIPTION
**Pull request by eternity74**
We can build libgapii.so for arm linux target with this patch.
To build libgapii.so, please follow below steps
 1. set proper path for sysroot and compiler in cmake/Toolchain/ArmLinux.cmake
    (see ARMLINUX_SYSROOT and ARMLINUX_COMPILER_BIN_PATH)
 2. ./do config
    'true' at "Build additional gapii for armlinx [Default: false]"

libgapii.so will be created at release/bin/armlinux

### CODE REVIEW

**Comment by eternity74** [comment]
I signed it!

File: cmd/do/env.go [hunk 1]
```diff
@@ -57,8 +57,8 @@ func env(cfg Config) *shell.Env {
 
 	path = append(path,
 		cfg.bin().System(),
-		cfg.JavaHome.Join("bin").System(),
 	)
+	path = append([]string{cfg.JavaHome.Join("bin").System()}, path...)
```
Comment (ben-clayton) [inline cmd/do/env.go]: What was the reason for this change?

Comment (eternity74) [inline cmd/do/env.go]: I have multiple java version on build system and java 1.7 was used basically.
And I set java 1.8 installation path during ./do config but during the build, java 1.7 was used bacause of /usr/bin/java which is version 1.7
If you prefer keep original code, I can revert this change.
Thanks.

Comment (ben-clayton) [inline cmd/do/env.go]: > If you prefer keep original code, I can revert this change.

Yes please, it may have unexpected side-effects in our build.

Comment (eternity74) [inline cmd/do/env.go]: I revert this change.
Thanks.

File: cmd/do/config.go [hunk 2]
```diff
@@ -63,6 +63,7 @@ type Config struct {
 	NinjaPath      file.Path `desc:"Path to ninja executable"`
 	PythonPath     file.Path `desc:"Path to python executable"`
 	MSYS2Path      file.Path `desc:"Path to msys2 root" os:"windows"`
+	ArmLinuxGapii  bool      `desc:"Build additional gapii for armlinx"`
```
Comment (ben-clayton) [inline cmd/do/config.go]: nit: s/armlinx/armlinux

Comment (eternity74) [inline cmd/do/config.go]: done. I uploaded change.

**Comment by AWoloszyn** [comment]
I rebased and double-checked this. Everything still seems to be working. Merging it.
